### PR TITLE
Update SMC Members

### PR DESCRIPTION
**Pull request by sharizahMA**
Updated the Treasurer information

### CODE REVIEW

File: _about-shss/School Management Committee.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ SCHOOL MANAGEMENT COMMITTEE (SMC)
 | Member     | Rev Victor Teo | Chaplain for SHPS and SHSS/Vicar, Chapel of Christ the Redeemer (CCR)     |
 | Member     |Ms Marilyn Ng     | Alumni President     |
 | Member     | Ms Irene Lim     | Parent Support Group (PSG) Chairman, SHPS     |
```
Comment (loyweejin) [inline _about-shss/School Management Committee.md:19]: Change Ms Irene Lim to Mr Seah Eng Wee

File: _about-shss/School Management Committee.md [hunk 2]
```diff
@@ -17,9 +17,9 @@ SCHOOL MANAGEMENT COMMITTEE (SMC)
...
-| Member     | Mr Shawn Teng     | PSG Chairman, SHSS     |
+| Member     | Mr Mark Joel Premraj     | PSG Chairman, SHSS     |
 | Member     | Mdm Lily John     | SHC Parochial Church Council (PCC) Representative    |
 | Member     | Mr Albert Seah     |CCR PCC Representative     |
```
Comment (loyweejin) [inline _about-shss/School Management Committee.md:22]: Remove Mr Albert Seah

**Review by loyweejin — CHANGES_REQUESTED** [review]

File: _about-shss/School Management Committee.md (hunk 1 above)
Comment (sharizahMA) [reply _about-shss/School Management Committee.md:19]: Ok  will do.

File: _about-shss/School Management Committee.md (hunk 2 above)
Comment (sharizahMA) [reply _about-shss/School Management Committee.md:22]: Completed.

**Review by loyweejin — APPROVED** [review]

**Pull request merged**